src/format/parser.d

```diff
@@ -2503,6 +2511,18 @@ private:
 		assert(0);
 	}
 
+	void parseShortenedFunctionBody() in(match(TokenType.FatArrow)) {
+		auto spanGuard = span!Span();
+		split();
+		nextToken();
+		split(true);
+		space();
+		parseExpression();
+		split();
```

Comment:
I don't think we want a split here either.

Reply:
gone